### PR TITLE
Web3picker redesign

### PR DESCRIPTION
**Pull request by JustCaptcha**
# Summary
Done: 
- Selecting provider
- Styling for a different amount of providers

Todo:
- Exit functionality
- Getting a provider image
- Add font-family according to the design

### CODE REVIEW

File: universal-login-web3/src/ui/react/Web3ProviderExtract.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  const pickLogo = (): string | undefined => {
+    switch (name) {
+      case 'UniversalLogin': return UniLoginLogo;
+    }
+  };
+
+  const logoImage = pickLogo();
+
+  return (
+    <ProviderButton name={name} selectedProvider={selectedProvider} onClick={ () => onClick()}>
```
Comment (alexcherman) [inline universal-login-web3/src/ui/react/Web3ProviderExtract.tsx]: You can write like this `onClick={onClick}`

File: universal-login-web3/src/ui/react/Web3PickerComponent.tsx [hunk 2]
```diff
@@ -1,37 +1,121 @@
...
+            <Web3ProviderExtract selectedProvider={selectedProvider} name={name} icon={icon} key={name} onClick={() => setSelectedProvider(name)}/>,
+          )}
+        </ProviderList>
+        <LoginButton onClick={() => setProvider(selectedProvider)}>Login</LoginButton>
+      </ModalBox>
     </ModalWrapper>
     : null;
 };
+
+const ButtonExit = styled.button`
```
Comment (alexcherman) [inline universal-login-web3/src/ui/react/Web3PickerComponent.tsx]: Use `<CloseButton />` component from common folder

File: universal-login-web3/src/ui/react/Web3PickerComponent.tsx [hunk 3]
```diff
@@ -1,37 +1,121 @@
...
+    height: 100%;
+    background-color: inherit;
+    transform: rotate(90deg);
+    position: absolute;
+    top: 0;
+    left: 0;
+  }
+`;
+
+const ModalTitle = styled.h3`
```
Comment (alexcherman) [inline universal-login-web3/src/ui/react/Web3PickerComponent.tsx]: Use `<Title />`component from common folder

File: universal-login-web3/src/ui/react/Web3PickerComponent.tsx [hunk 4]
```diff
@@ -1,37 +1,121 @@
...
+  font-size: 2.4rem;
+  text-align: center;
+  color: #0F0C4A;
+`;
+
+const ModalText = styled.p`
```
Comment (alexcherman) [inline universal-login-web3/src/ui/react/Web3PickerComponent.tsx]: Use `<Text />`component from common folder

File: universal-login-web3/src/ui/react/Web3PickerComponent.tsx [hunk 5]
```diff
@@ -1,37 +1,121 @@
...
+  min-height: 30rem;
+  color: white;
+  color: #0F0C4A;
+  background-color: #fff;
+  padding: 4.6rem 4.4rem 4rem;
+  border-radius: .8rem;
+  position: relative;
+`;
+
+const LoginButton = styled.button`
```
Comment (alexcherman) [inline universal-login-web3/src/ui/react/Web3PickerComponent.tsx]: Use `<ButtonPrimary />`component from common folder

File: universal-login-web3/src/ui/react/playground/WaitForTransactionModalPlayground.tsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+export const WaitForTransactionModalPlayground = () => {
+  const [web3Strategy] = useState(() => new Web3Strategy([
+    universalLoginProviderFactory,
+  ],
+  {} as Provider,
+  ));
+  const [providerName, setSetProviderName] = useState<string>(web3Strategy.providerName);
+
+  const onClick = async () => {
+    console.log('clicked button');
```
Comment (JustynaBroniszewska) [inline universal-login-web3/src/ui/react/playground/WaitForTransactionModalPlayground.tsx]: Remove console.logs

File: universal-login-web3/src/ui/react/playground/WaitForTransactionModalPlayground.tsx [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    await web3Strategy.send({method: 'eth_sign'} as any, () => {});
+    console.log('Picked');
```
Comment (JustynaBroniszewska) [inline universal-login-web3/src/ui/react/playground/WaitForTransactionModalPlayground.tsx]: remove

File: universal-login-web3/src/ui/react/Web3PickerComponent.tsx [hunk 8]
```diff
@@ -1,37 +1,93 @@
...
+import {ButtonPrimary} from './common/Button/Button';
 
-interface Web3PickerComponentProps {
+interface IWeb3PickerComponentProps {
   isVisibleProp: Property<boolean>;
   hideModal: () => void;
   factories: Web3ProviderFactory[];
-  setProvider: (providerName: string) => Promise<void>;
+  setProvider: (providerName: string) => void;
+
```
Comment (marekkirejczyk) [inline universal-login-web3/src/ui/react/Web3PickerComponent.tsx]: unnecessary newline

File: universal-login-web3/src/ui/react/Web3ProviderExtract.tsx [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+interface IIndicatorProps {
+  status: boolean;
+}
+
+interface IProviderButton {
+  name: string;
+  selectedProvider: string;
+}
+
+const Web3ProviderExtract = ({name, icon, labelTop, labelBottom, indicator, selectedProvider, onClick}: IWeb3ProviderProps) => {
```
Comment (marekkirejczyk) [inline universal-login-web3/src/ui/react/Web3ProviderExtract.tsx]: `<ProviderSelector />`

**Comment by JustCaptcha** [comment]
image

**Comment by JustCaptcha** [comment]
image

**Pull request merged**